scylla-cdc-base/src/test/java/com/scylladb/cdc/model/worker/MockRawChange.java

```diff
@@ -46,6 +50,13 @@ public Builder withStreamId(ByteBuffer streamId) {
             return this;
         }
 
+        public Builder withStreamId(GenerationMetadata generationMetadata, int vnodeNumber, int streamIndex) {
```

Comment:
I won't insist but I'd enjoy a more descriptive name for `streamIndex`. Like `streamIndexWithinVNode...`

Reply:
Fixed in the amended commit.